### ISSUE TITLE
Errors occur under certain conditions using payment sheet and custom flow.

### ISSUE THREAD

**Issue by sakyoyuto** [flutter-stripe/flutter_stripe#1422]
Hello.
When using the custom flow in paymentSheet, an error will occur if there is no payment information associated with the customer.
This will not occur if customFlow is set to false or if the customer already has a payment method associated with the customer.

The errors thrown in such cases are as follows
```
StripeError<PaymentSheetError>(message: Unknown result this is likely a problem in the plugin {paymentOption: null}, code: PaymentSheetError.unknown)
```

Example code.
```
await Stripe.instance.initPaymentSheet(
        paymentSheetParameters: SetupPaymentSheetParameters(
          customFlow: true,
          merchantDisplayName: 'hello',
          paymentIntentClientSecret: paymentData.client_secret,
          customerEphemeralKeySecret: paymentData.ephemeral_key,
          customerId: paymentData.customer_id,
          applePay: null,
          googlePay: null,
        ),
      );
      final res = await Stripe.instance.presentPaymentSheet();
```

Using flutter_stripe 9.4.0 and flutter 3.13.4.

Below doctor
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.13.4, on macOS 13.5.2 22G91 darwin-arm64, locale ja-JP)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.82.2)
[✓] Connected device (3 available)
[✓] Network resources

• No issues found!
```

Thank you.

**Comment by sakyoyuto** [comment]
I'm sorry.
It was in #1364.

**Comment by appeventic** [comment]
hey @sakyoyuto how did you resolve this issue? I tried to use  version 9.4.0 but still facing same issue?

**Comment by sakyoyuto** [comment]
Hi @appeventic 
This problem was not resolved in version 9.4.0. I gave up using customflow.

**Comment by appeventic** [comment]
> Hi @appeventic This problem was not resolved in version 9.4.0. I gave up using customflow.

Thanks for your reply. Did you face this problem in both iOS and Android?
